### DISCUSSION TITLE
Disable wintercms frontend (to use only backend)

### DISCUSSION THREAD

**Discussion by YannPl** [wintercms/winter#205] [Q&A] [answered]
I'm openning this discussion following the chat I had withn @LukeTowers in the discord.

I use wintercms for it's backend webapp possibilities and want to disable the frontend and redirect automatically to the backend interface.

The solution is quite simple, in the config:
```
- cms.backendUri = ''
- cms.loadModules = ['System', 'Backend'] // remove 'Cms'
```

You can then drop the cms requirement from composer.json

**Comment by bennothommo** [comment]
@YannPl would you be interested in submitting a PR to the [docs repo](https://github.com/wintercms/docs) detailing this setup? I think it would be great to make it available in the main docs and show that Winter CMS can build more than just websites.

**Comment by bennothommo** [comment, marked as answer]
To clarify on @YannPl's answer - if you wish to run Winter CMS with only the Backend (ie. as a self-contained application), you can make the following changes to `config/cms.php`:

```php
'backendUri' => '', // Set to an empty string to use the root domain or subdomain.

'loadModules' => ['System', 'Backend'], // Remove 'Cms' to not use the CMS module.
```

In addition, you can remove the following line from the `require` block in `composer.json` to not download the CMS module or any updates to it:

```php
"winter/wn-cms-module": "<version>",
```

Accessing your root domain or subdomain will result in the Backend login screen, or Dashboard if you are logged in, showing on load.
